### DISCUSSION TITLE
`*` import

### DISCUSSION THREAD

**Discussion by elouin** [hylang/hy#2350] [Q&A] [answered]
Hi,

I am trying to basically do `from pyray import *`, so i tried `(import pyray [*])`, but i get the following error:
`ImportError: cannot import name 'hyx_XasteriskX' from 'pyray'`

The example `(import hy.model-patterns [*])` from the [docs](https://docs.hylang.org/en/master/model_patterns.html#a-motivating-example) gives me the same error.

I feel like i am doing something obvious wrong here.

**Comment by Kodiologist** [comment, marked as answer]
The syntax now looks like `(import pyray *)`, but I failed to update that example in the documentation. Sorry about that. The documentation of `import` itself is up to date, though.

**Reply by elouin** [reply]
Ahh, i see. Thanks. Is that only for asterisk imports or imports in general?

**Reply by Kodiologist** [reply]
Have you read the documentation of `import`?

**Reply by elouin** [reply]
Now i read https://docs.hylang.org/en/master/api.html#import ;) Thanks for your help.
